### PR TITLE
Move repo from croaky/parity to thoughtbot/parity

### PR DESCRIPTION
**Pull request by croaky**

### CODE REVIEW

**Comment by jsteiner** [comment]
Aww yeah! Do we want to add the [About thoughtbot](https://github.com/thoughtbot/factory_girl#about-thoughtbot) section as well?

**Comment by croaky** [comment]
@jsteiner Good call, added. Ready for another review.

File: README.md [hunk 1]
```diff
@@ -156,11 +159,26 @@ Releasing
 
 See guidelines in RELEASING.md for details
 
-Credits
+License
 -------
 
-Parity is maintained by Dan Croak. It is free software and may be redistributed
-under the terms specified in the LICENSE file.
+Laptop is © 2013-2015 thoughtbot, inc.
```
Comment (jsteiner) [inline README.md]: Parity

**Comment by jsteiner** [comment]
One comment, otherwise LGTM

File: README.md (hunk 1 above)
Comment (croaky) [inline README.md]: D'oh. Thanks, fixed.

**Pull request merged**